### DISCUSSION TITLE
Macie job details attribute isMonitoredByJob is not listed in create-classification-job

### DISCUSSION THREAD

**Discussion by achavan-cr** [aws/aws-cli#8144] [Q&A]
Hi everyone,

$ aws macie2 describe-buckets     --criteria '{"bucketName":{"prefix":"BUCKETNAME"}}'

```
{
    "buckets": [
        {
            "accountId": "xxxxxxxxxx",
            "allowsUnencryptedObjectUploads": "TRUE",
            "bucketArn": "arn:aws:s3:::BUCKETNAME",
            "bucketCreatedAt": "2023-08-17T13:51:21+00:00",
            "bucketName": "BUCKETNAME",
            "classifiableObjectCount": 1,
            "classifiableSizeInBytes": 19,
            "jobDetails": {
                "isDefinedInJob": "TRUE",
                "isMonitoredByJob": "FALSE",
                "lastJobId": "xxxxxxxxxxxxxxxxxxxxxx",
                "lastJobRunTime": "2023-08-28T07:16:20.090000+00:00"
            },
            "lastAutomatedDiscoveryTime": "2023-08-25T11:03:49.633000+00:00",
            "lastUpdated": "2023-08-28T08:45:49.298000+00:00",
......
}
```

here, we want to update this attribute **isMonitoredByJob** to True but we dont have option to update existing job.

Even while creating new job using https://docs.aws.amazon.com/cli/latest/reference/macie2/create-classification-job.html we do n't have this attribute mentioned in the sample json given in this document.

**Comment by aBurmeseDev** [comment]
Hi @achavan-cr - thanks for reaching out. This sounds like a feature request for the Service team and I'm reaching out to them on you behalf to check if there's a way to update `isMonitoredByJob`. Please check back in for an update.

**Comment by aBurmeseDev** [comment]
@achavan-cr - I just heard back from a service team member and was advised that the field `isMonitoredByJob` is not mutable by the users - It's an internal state which is reported back as part of `describe-buckets` and `jobDetails` but it's only set internally. Basically the API returns the boolean if there's a scheduled job against the bucket which will be true and otherwise false. 

Hope that helps!
